content/community/blm.rst

```diff
@@ -0,0 +1,149 @@
+public: yes
+location: community
+grid_template: true
+hide_contact: true
+headline:
+  - tagline: "Donate to the cause, and we'll match you"
+    type: 'Community'
+image:
+  - src: 'pages/blm.jpg'
+summary: |
+  **OddBird stands with the Black community
+  in grieving and protesting racist police violence and murder.**
+  We must all demand an end to
+  the state-sanctioned assault on Black people,
+  and take action to confront
+  anti-Blackness in all our systems.
+  There is much to do, but one action is simple and clear:
+  we want to support the organizations
+  already working in the streets and behind the scenes --
+  and **we're inviting you to join us**.
+orgs:
+  - title: 'Black Lives Matter & their "Places To Donate"'
+    url: 'https://blacklivesmatters.carrd.co/'
+    icon: 'fist'
+    text: |
+      #BlackLivesMatter was founded in 2013
+      in response to the acquittal of Trayvon Martin’s murderer.
+      Their mission is to eradicate white supremacy
+      and build local power to intervene in violence
+      inflicted on Black communities by the state and vigilantes.
+      We'll match direct donations to BLM
+      or anyone they list on their donation page.
+
+      .. callmacro:: content.macros.j2#link_button
+        :url: 'https://blacklivesmatters.carrd.co/#donate'
+        :content: 'Donate through Black Lives Matter'
+
+  - title: Black Visions Collective
+    url: 'https://www.blackvisionsmn.org/'
+    icon: 'fist'
```

Comment:
Is it intentional to use the fist for all these, or were these going to be the individual org logos?

Reply:
It was intentional - I'm open to adding org logos, but didn't feel like it was important to hold up the branch for it, especially since I'm not sure they all _have_ logos.